spec/controllers/comments_controller_spec.rb

```diff
@@ -2,6 +2,23 @@
 
 RSpec.describe CommentsController, type: :controller do
 
+  it "assigns a newly created comments as comment" do
+    @user = User.create!(email: "test@test.com", password: 'testtest')
```

Comment:
Presumably you're doing this because a user has to be signed in.  Is there a way that you can sign in a user, rather than doing this stub?